lib/utils/isErrorObject.ts

```diff
@@ -1,3 +1,3 @@
-module.exports = function(err) {
+export = function(err: any) {
```

Comment:
I used `any` here because that's exactly what we are testing in the function, if the `err` argument is an error or not, so the argument can be anything and it will return `true` or `false` based on whether the `err` argument is an `Error` instance of not.